content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,1141 @@
+
+
+You have successfully provisioned an Edge cluster and deployed the Hello Universe application on it. 
+
+<br />
+
+# Cleanup
+
+The following steps will guide you in cleaning up your environment, including the cluster, cluster profile, and Edge hosts. Follow the steps for Palette if you used Palette to deploy the cluster. Use Terraform commands to delete the cluster if you used Terraform for deployment. 
+<br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Palette UI" key="palette_ui_delete">
+
+<br />
+
+##  Delete the Cluster and Profile using Palette UI
+
+To delete the cluster, view the cluster details page. Click on the **Settings** button to expand the **drop-down Menu**, and select the **Delete Cluster** option, as shown in the screenshot below
+
+
+![Screenshot of deleting a cluster.](/tutorials/edge/clusters_edge_deploy-cluster_delete-cluster.png)
+
+
+Palette will prompt you to enter the cluster name and confirm the delete action. Type the cluster name to proceed with the delete step. The cluster status will turn to **Deleting**. Deletion takes up to 10 minutes.
+
+
+After you delete the cluster, click **Profiles** on the left **Main Menu**, and select the profile to delete. Choose the **Delete** option in the **three-dot Menu**, as shown in the screenshot below.
+
+
+![Screenshot of deleting a cluster profile.](/tutorials/edge/clusters_edge_deploy-cluster_delete-profile.png)
+
+
+Please wait until Palette deletes the resources successfully. 
+<br />
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="Terraform" key="terraform_ui_delete">
+
+<br />
+
+##  Delete the Cluster and Profile using Terraform
+
+Switch back to the tutorials container. If you have closed the terminal session, you can reopen another bash session in the tutorials container using the following command. 
+<br />
+
+```bash
+docker exec -it tutorialContainer bash
+``` 
+
+Issue the following command from within the **/terraform/edge-tf** directory.
+<br />
+
+```bash
+terraform destroy -auto-approve
+```
+
+Wait for the resources to clean up. Deleting the Terraform resources may take up to 10 minutes. 
+<br />
+
+```bash hideClipboard
+# Output condensed for readability
+Destroy complete! Resources: 2 destroyed.
+```
+
+Exit from the tutorials container to come back to the Linux development environment.
+<br />
+
+```bash
+exit
+```
+
+
+<br />
+
+</Tabs.TabPane>
+
+</Tabs>
+
+##  Delete Edge Hosts
+
+Switch back to the **CanvOS** directory in the Linux development environment containing the **.goenv** file, and use the following command to delete the Edge hosts. 
+<br />
+
+```bash
+docker run -it --rm --env-file .goenv \
+  ghcr.io/spectrocloud/tutorials:1.0.6 \
+  sh -c "cd edge/vmware/clone_vm_template/ && ./delete-edge-host.sh"
+```
+
+Also, delete the **palette-edge-installer.iso** file from the **packer_cache/** directory in the VMWare vCenter datastore.
+
+##  Delete Edge Artifacts
+
```

Comment:
We also need to guide the user in cleaning up the VM template and the uploaded ISO from their folder. This adds up, so make sure you have deleted your templates and ISOs as well.